### PR TITLE
[16.0] [mig] analytic_base_department

### PR DESCRIPTION
**Pull request by aktiv-heli-kantawala**

### CODE REVIEW

**Review by misern2 — APPROVED** [review]
LGTM :+1:

File: analytic_base_department/models/analytic.py [hunk 1]
```diff
@@ -21,8 +20,10 @@ class AnalyticLine(models.Model):
     def _default_department(self):
         department_id = False
         employee = self.env.user.employee_ids
+        print("=============employeeee---------", employee)
```
Comment (rousseldenis) [inline analytic_base_department/models/analytic.py]: @heliaktiv Could you remove those traces ?

File: analytic_base_department/tests/test_line_department.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
-# Copyright 2016 Yannick Vaucher (Camptocamp)
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+# -*- coding: utf-8 -*-
```
Comment (dreispt) [inline analytic_base_department/tests/test_line_department.py]: Why are you removing copyright statements?
The utf-8 marker on Python files is redundant on Python3

**Pull request closed without merging**